src/ast/ast.c

```diff
@@ -359,6 +358,10 @@ int TraverseRecordCap(const AST *ast) {
 	return MIN(ast->limit, 16);  // Use 16 as the default value.
 }
 
+AST_AnnotationCtxCollection *AST_GetAnnotationCtxCollection(AST *ast) {
```

Comment:
All of the annotation getters should be static (if possible) and inlined.

Reply:
cannot be static. inline